api/src/services/assets.ts

```diff
@@ -168,7 +168,7 @@ export class AssetsService {
 				});
 			}
 
-			const readStream = await storage.location(file.storage).read(file.filename_disk, range);
+			const readStream = await storage.location(file.storage).read(file.filename_disk, { range, bustCache: true });
```

Comment:
It feels a little tricky to pass `bustCache` here, as we know that most drivers ignore it 🤔 Seeing this is only required for Cloudinary, maybe we add it in just there?

Reply:
Alternatively, maybe we can pass the `last_updated` field value from `directus_files` as an optional parameter and use that as a cache buster just in cloudinary? 🤔